src/bzflag/InputMenu.cxx

```diff
@@ -279,15 +288,26 @@ void            InputMenu::callback(HUDuiControl* w, const void* data)
     /* Grab mouse */
     case 'G':
     {
-        const bool grabbing = (selectedOption == "Window");
+        const int index = listHUD->getIndex();
+        // Window
+        const bool grabbing = (index == 1);
         BZDB.set("mousegrab", grabbing ? "1" : "0");
         getMainWindow()->enableGrabMouse(grabbing);
 
-        const bool clamped = (selectedOption == "MotionBox");
+        // Mouse Box
+        const bool clamped = (index == 2);
         BZDB.set("mouseClamp", clamped ? "1" : "0");
     }
     break;
 
+
+    case 'M':
+    {
+        SceneRenderer* renderer = getSceneRenderer();
+        renderer->setMaxMotionFactor(listHUD->getIndex() - 11);
```

Comment:
elsewhere, `renderer` is tested for `nullptr` before use. Is it guaranteed not to be null here?